docs/resources/Channel.md

```diff
@@ -195,9 +195,10 @@ Represents a message sent in a channel within Discord.
 | webhook\_id? | snowflake | if the message is generated by a webhook, this is the webhook's id |
 | type | integer | [type of message](#DOCS_RESOURCES_CHANNEL/message-object-message-types) |
 | activity? | [message activity](#DOCS_RESOURCES_CHANNEL/message-object-message-activity-structure) object | sent with Rich Presence-related chat embeds |
-| application? | [message application](#DOCS_RESOURCES_CHANNEL/message-object-message-application-structure) object | sent with Rich Presence-related chat embeds | |
+| application? | [message application](#DOCS_RESOURCES_CHANNEL/message-object-message-application-structure) object | sent with Rich Presence-related chat embeds |
+| flags | [integer bitfield](#DOCS_RESOURCES_CHANNEL/message-object-message-flags) | bitfield of multiple boolean flags |
```

Comment:
The flag property isn't on the message object _unless_ embeds are suppressed right now. Should be marked as optional (`flags?`)

Reply:
From my testing, the flag was `0` on messages not being suppressed. Gonna test again tho

Reply:
I don't seem to get the field when GETing from REST... Which is why I still believe it should be marked as optional, unless it's an API bug/inconsistency